pcbuilder_crawlers/templates/video_card.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": ["Productnaam"],
+	"subname": [],
+	"info": [],
+	"stock": [],
+	"image": [],
+	"ean": ["EAN code"],
+	"sku": [],
+	"SKU": [],
+	"link": [],
+	"brand": ["Merk"],
+	"execution": [],
+	"memory_specification": [],
+	"serie": [],
+	"product": [],
+	"memory_size": [],
+	"number": [],
+	"module_size": [],
+	"Price_per_gb": [],
+	"memory_type": [],
```

Comment:
ah ja oke je kan het wel
```
"ean" : ["EAN code"]
```
laten dan. Maar we hebben nog wel veel te veel rows aan lege arrays die we niet willen vertalen